app/src/main/java/ch/epfl/cs311/wanderwave/navigation/NavigationActions.kt

```diff
@@ -59,6 +67,13 @@ class NavigationActions(navController: NavHostController) {
   fun goBack() {
     navigationController.popBackStack()
   }
+
+  fun signIn() {
```

Comment:
If we just navigateToTopLevel the stack isn't cleared so user can manually go back to Login. The only possible way to navigate to Login must be through sign out button.

Reply:
Shouldn't that be the behaviour when navigating to all topLevelDestionations? It indeed looks like it doesn't behave like this at the moment, but I would consider that a bug of the `navigateToTopLevel` implementation.

Reply:
Yeah I am not so sure for the moment, maybe we should consider refactoring navigation. And I wanted this behavior because after signing in and presented on the mainscreen when user clicks back button of the phone the app should minimize instead of navigating back because mainscreen is at the index 0 of the navigation stack after logging in. I think it behaves like so for the moment.